src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig

```diff
@@ -1,3 +1,13 @@
+<div>
+{% spaceless %}
+    <span class="toggle-icon">
+        <a href="#" onclick="toggle('data-{{ prefix ~ '-' ~ i }}'); switchIcons('icon-{{ prefix ~ '-' ~ i }}-open', 'icon-{{ prefix ~ '-' ~ i }}-close'); return false;">
```

Comment:
if/when this gets merged, will people mind a PR that switches out the onclick methods and such for the class here that can be handled in in script tag included in the main layout file?  That way I don't have to keep full copies of these templates simply to please my CSP policy.